salt/loader/__init__.py

```diff
@@ -146,7 +146,7 @@ def _module_dirs(
     ext_type_types = []
     if ext_dirs:
         if ext_type_dirs is None:
-            ext_type_dirs = "{}_dirs".format(tag)
+            ext_type_dirs = f"{tag}_dirs"
```

Comment:
Annoyingly pre-commit has made a mess of this patch including unrelated changes.
It would be better to pyupgrade/black/isort the whole repo in a single commit so contributor patches can be as clean as possible

Reply:
I agree this could be done in a bulk or several smaller PRs. That said, for now you could also separate the your changes from pre-commit's in two separate commits. I think `--no-verify` can accomplish that.